recipes/python_http_client/meta.yaml

```diff
@@ -1,44 +0,0 @@
-{% set name = "python_http_client" %}
```

Comment:
There is no need to remove lingering recipes. But in this case it does not hurt b/c those are already feedstocked.